packages/yoga/src/Button/Button.theme.js

```diff
@@ -124,6 +124,20 @@ const Button = ({
         bottom: spacing.xxsmall,
       },
     },
+    icon: {
+      size: {
+        default: 24,
```

Comment:
Why do not use `spacing.large` or `'large'` (depending on the context) instead of `24`, it's also not necessary once we are defining the icon size and its margin.